node/test/mocktests.ts

```diff
@@ -183,5 +184,53 @@ describe('Mock Tests', function () {
         
         assert(tool, "tool should not be null");
         assert(rc == 0, "rc is 0");
-    })                
+    })        
+    
+    it('Mock toolRunner returns correct output', async () => {
+        const expectedStdout = "atool output here" + os.EOL + "abc";
```

Comment:
optional: You could add a separate test for trailing newline, e.g. `const expectedStdout = os.EOL`